homework/Contributor.js

```diff
@@ -13,7 +13,37 @@ class Contributor {
    * @param {HTMLElement} container The container element in which to render the contributor.
    */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
+    // Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
+    const contributorsDiv = Util.createAndAppend('div', container);
+
+    const contributorsLists = Util.createAndAppend('ul', contributorsDiv, { id: 'conts-list' });
+
+    const contributorListItem = Util.createAndAppend('li', contributorsLists, {
+      id: `${this.contributor.login}-list-item`,
+      class: 'cont-li',
+    });
+
+    Util.createAndAppend('img', contributorListItem, {
+      id: `${this.contributor.login}Img`,
+      class: 'cont-avatar',
+      src: this.contributor.avatar_url,
+    });
+
+    const contributorDiv = Util.createAndAppend('div', contributorListItem, {
+      id: `${this.contributor.login}Div`,
+    });
+
+    Util.createAndAppend('a', contributorDiv, {
+      id: `${this.contributor.login}Name`,
+      class: 'cont-name',
+      text: this.contributor.login,
+      href: this.contributor.html_url,
+      target: '_blank',
+    });
+    Util.createAndAppend('div', contributorDiv, {
+      id: `${this.contributor.login}Badge`,
```

Comment:
you are almost using an `id` on every element you have, `id` we used to use them mostly in Css.

but even in Css we had a note about overusing it, it's good when we have a use case for it.
Also when we don't have a one, we are having an extra `id` I may use it's same value in other place so the browser gets confused over which he have to handle.

Reply:
Actually I have no idea why I used id for every element. So I removed them.